config/routes.rb

```diff
@@ -197,48 +197,8 @@
 
     resources :maintenance_windows, only: [:index]
 
-    resources :prescriptions, only: %i[index show], defaults: { format: :json } do
-      get :active, to: 'prescriptions#index', on: :collection, defaults: { refill_status: 'active' }
-      patch :refill, to: 'prescriptions#refill', on: :member
-      resources :trackings, only: :index, controller: :trackings
-      collection do
-        resource :preferences, only: %i[show update], controller: 'prescription_preferences'
-      end
-    end
-
-    resource :health_records, only: [:create], defaults: { format: :json } do
-      get :refresh, to: 'health_records#refresh', on: :collection
-      get :eligible_data_classes, to: 'health_records#eligible_data_classes', on: :collection
-      get :show, controller: 'health_record_contents', on: :collection
-    end
-
     resources :appeals, only: :index
 
-    scope :messaging do
```

Comment:
I see many controllers associated with these routes that are still around. They should be removed too, right?